.appveyor.yml

```diff
@@ -62,6 +62,14 @@ install:
       If (!(Test-Path cpdksetup.exe )) {
           appveyor DownloadFile "https://download.microsoft.com/download/1/7/6/176909B0-50F2-4DF3-B29B-830A17EA7E38/CPDK_RELEASE_UPDATE/cpdksetup.exe"
       }
+  - wget https://ftp.gnu.org/gnu/gengetopt/gengetopt-2.23.tar.xz
+  - tar xJf gengetopt-2.23.tar.xz
+  - cd gengetopt-2.23
+  - bash -c "exec 0</dev/null && ./configure --prefix=/ || cat config.log"
+  - bash -c "exec 0</dev/null && make"
+  - cd ..
+    #- curl -sSf -o C:\cygwin64\setup-x86_64.exe https://cygwin.com/setup-x86_64.exe
+    #- C:\cygwin64\setup-x86_64.exe -q -P gengetopt
```

Comment:
I see this was likely an attempt to install it through cygwin. Is it still needed or is is just an artifact of failed attempt to avoid people trying? I would propose to remove the commented out lines or added some explanation why it did not work.